test_conformance/math_brute_force/binary_operator_float.cpp

```diff
@@ -688,6 +720,9 @@ int TestFunc_Float_Float_Float_Operator(const Func *f, MTdata d,
     double maxErrorVal = 0.0;
     double maxErrorVal2 = 0.0;
 
+    // reciprocal differs from divide only in relaxed mode, skip otherwise
+    if ((strcmp(f->name, "reciprocal") == 0) && !relaxedMode) return CL_SUCCESS;
```

Comment:
This is true currently, but it might not be true for long, see: https://github.com/KhronosGroup/OpenCL-Docs/pull/1293

I think it would be better to test "reciprocal" in all cases, even those where the required accuracy matches "divide".

Reply:
@bashbaug I can't spot any plans to change spec in reference to fp32 reciprocal precision, am I missing something ?

Reply:
Discussed in the January 21st teleconference.  We would prefer to test all of the cases, even if there may be some overlap with the divide test:

1. In theory, "divide" and "reciprocal" may have different accuracy requirements, though we are currently trending to keep them in sync.
2. Testing "reciprocal" explicitly will give us more coverage for the "1.0 / x" cases explicitly, which we may not get through "divide".

Reply:
Thanks, reciprocal activated for all fp16/fp32/fp64 and relaxed cases